### PR TITLE
[EN/FR] Update "Timelines" and "Timing"; add updated screenshots

### PR DESCRIPTION
**Pull request by DoryanR**
Update of the translations of the following wiki pages:

- `Beatmap_Editor/Timelines`
- `Beatmap_Editor/Timing`

Also:

- Addition of the French images for both articles
- Update images in `Beatmap_Editor/Timing`, and fix links in othens articles.

This PR is the last part of the other PRs concerning beatmap editor.

### CODE REVIEW

**Review by AirOne01 — DISMISSED** [review]
Ok ! 👍

File: wiki/Beatmap_Editor/Timing/fr.md [hunk 1]
```diff
@@ -1,23 +1,25 @@
----
-outdated: true
----
+# Onglet Timing 
 
-# Onglet Timing
+*Voir aussi : [How to time songs](/wiki/Guides/How_to_Time_Songs)*
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Timing/fr.md:3]: Il faut une traduction pour  les crochets mais j'ai pas d'autre traduction qui me vient à l'esprit à part remplacer "synchroniser" par "timer".
```suggestion
*Voir aussi : [Comment synchroniser les musiques](/wiki/Guides/How_to_Time_Songs)*
```

File: wiki/Beatmap_Editor/Timing/fr.md [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
 
-Un timing correct est impératif dans le processus de beatmapping pour permettre de placer les objets à temps avec le rythme.
+## Fenêtre Timing et Control Points
 
-## Setup
+![Screenshot de la fenêtre Timing et Control Points](img/TimingSetup.png)
 
-![Capture d'écran de la fenêtre "Chronométrage et points de contrôle"](img/TimingSetup.jpg "Le dialogue de configuration du timing montre les fluctuations du BPM de la map et les changements de jeu d'échantillons")
+Les points de timing peuvent être ajoutés à partir de la fenêtre `Timing and Control Points` de l'éditeur (raccourci par défaut `F6`). Les points de synchronisation divisent un morceau en sections de synchronisation pour refléter précisément les changements de synchronisation d'un morceau.
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Timing/fr.md:15]: Est-ce que points de timing = points de synchronisation ? Si oui, autant choisir une forme et tout unifier pour éviter toute confusion.

**Review by Satsukel — CHANGES_REQUESTED** [review]

File: wiki/Beatmap_Editor/Timing/fr.md (hunk 2 above)
Comment (DoryanR) [inline wiki/Beatmap_Editor/Timing/fr.md:15]:
```suggestion
Les points de timing peuvent être ajoutés à partir de la fenêtre `Timing and Control Points` de l'éditeur (raccourci par défaut `F6`). Les points de timing divisent un morceau en sections de synchronisation pour refléter précisément les changements de timing d'un morceau.
```

**Review by Satsukel — APPROVED** [review]

**Review by TicClick — APPROVED** [review]
I've reverted `Timing_base.jpg` and its French version back to JPEG, as it shaves ~500kb off the article and does not harm the quality

**Comment by DoryanR** [comment]
> I've reverted `Timing_base.jpg` and its French version back to JPEG, as it shaves ~500kb off the article and does not harm the quality

Ok no problem

**Pull request merged**